src/main/java/org/broadinstitute/hellbender/utils/codecs/gtf/EnsemblGtfCodec.java

```diff
@@ -162,23 +203,27 @@ static boolean validateEnsemblGtfFeature(final GencodeGtfFeature feature) {
      */
     @VisibleForTesting
     boolean validateHeader(final List<String> header, final boolean throwIfInvalid) {
-        if ( header.size() != HEADER_NUM_LINES) {
-            if ( throwIfInvalid ) {
-                throw new UserException.MalformedFile(
-                        "ENSEMBL GTF Header is of unexpected length: " +
-                                header.size() + " != " + HEADER_NUM_LINES);
-            }
-            else {
-                return false;
+        // As it turns out, the ENSEMBL GTF header is pretty loosy-goosy.
+        // No fields are required, and therefore it could actually be empty.
```

Comment:
disgusting

Reply:
Yeah - according to the spec, you can get the version from the file names.

I plan on implementing this after the `NameAwareCodec` PR goes in.